rust/src/ffi.rs

```diff
@@ -1174,6 +1174,29 @@ impl TryFrom<ConfirmationsPolicy> for data_api::wallet::ConfirmationsPolicy {
     }
 }
 
+/// A single-use transparent address, along with metadata about the address's use within the
+/// wallet's ephemeral gap limit.
+#[repr(C)]
+pub struct SingleUseTaddr {
```

Comment:
Add an entry to `build.rs` renaming this to `FfiSingleUseTaddr` in Swift.

(Beyond the historical reasons, I try to ensure this for all new FFI types that contain pointers to raw Rust memory, to make it clear they must not be used beyond the FFI boundary.)